src/internal.c

```diff
@@ -5534,6 +5545,16 @@ static int DoCertificate(WOLFSSL* ssl, byte* input, word32* inOutIdx,
                             #endif /* HAVE_ECC */
                         #endif /*HAVE_PK_CALLBACKS */
                     }
+
+                    /* check size of peer ECC key */
+                    if (ret == 0 && ssl->peerEccDsaKeyPresent &&
+                                              !ssl->options.verifyNone &&
+                                              wc_ecc_size(ssl->peerEccDsaKey)
+                                              < ssl->options.minEccKeySz) {
```

Comment:
Isn't this comparing a signed value (wc_ecc_size) to an unsigned value (minEccKeySz)?  If so, that should be avoided.